runtime/onert/api/src/nnfw_api_internal.h

```diff
@@ -206,6 +210,7 @@ struct nnfw_session
   std::shared_ptr<onert::api::CustomKernelRegistry> _kernel_registry;
   std::vector<std::thread> _threads;
   uint32_t _training_step{0};
+  std::unique_ptr<onert::ir::train::TrainingInfo> _train_info;
```

Comment:
Q: Is there a plan to support multiple models in training features? 
     If so, Since the session hold multiple models through `_nnpkg`, It might be better to add a vector of Training info

```suggestion
  std::vector<std::unique_ptr<onert::ir::train::TrainingInfo>> _train_info;
```


---------------------------------------------------

I thought there no plan for support multiple model training at **near** future.
So, I'd like to save first model(primary model)'s TrainingInfo in the session.